modules/context/csrf.go

```diff
@@ -162,7 +163,12 @@ func prepareOptions(options []CsrfOptions) CsrfOptions {
 
 	// Defaults.
 	if len(opt.Secret) == 0 {
-		opt.Secret = string(com.RandomCreateBytes(10))
+		randBytes, err := util.CryptoRandomBytes(8)
```

Comment:
Just to note for other reviewers/maintainers that the reduction of this isn't a concern. 

Old situation:
The `com.RandomCreateBytes` actually doesn't give you the full 8 bits of randomness per byte(so 80 bits in the old case), because it reduces itself to only use the alphanum: https://github.com/unknwon/com/blob/b41c64acd94be7e673c9c8301344d31cce99e06c/string.go#L130 which only gives 6 bits per byte of randomness so it would only receive 48 bits of randomness.

New situation:
With `CryptoRandomBytes` the full 64 bits would be random.

Reply:
And before, IIRC the `com.RandomCreateBytes` uses a math PRNG ..... not crypto-safe.